components/dot_menu/dot_menu.tsx

```diff
@@ -224,15 +234,15 @@ export default class DotMenu extends React.PureComponent {
         </Tooltip>
     )
 
-    refCallback = (menuRef) => {
+    refCallback = (menuRef: any) => {
```

Comment:
Is there something we can use that is more strict that `any`? Perhaps `Menu` from `components/widgets/menu/menu.tsx`  can be used here?

Reply:
done in latest commit